x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/endpoint_action_response_codes.ts

```diff
@@ -123,6 +123,97 @@ const CODES = Object.freeze({
+    }
+  ),
+
+  // Dev:
+  // Executing timeout has been reached, the command was killed.
+  'ra_execute_error_processing-timeout': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.processingTimeout',
+    { defaultMessage: 'Command execution was terminated. It exceeded the provided timeout.' }
+  ),
+
+  // Dev:
+  // Execution was interrupted, for example: system shutdown, endpoint service stop/restart.
+  'ra_execute_error_processing-interrupted': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.processingInterrupted',
+    {
+      defaultMessage: 'Command execution was absolutely interrupted.',
+    }
+  ),
+
+  // Dev:
+  // Too many active execute actions, limit 10. Execute actions are allowed to run in parallel, we must
+  // take into account resource use impact on endpoint as customers are piky about CPU/MEM utilization.
+  'ra_execute_error_to-many-requests': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.toManyRequests',
+    {
+      defaultMessage: 'Too many concurrent command execution actions.',
+    }
+  ),
+
+  // Dev:
+  // generic failure (rare corner case, software bug, etc)
+  ra_execute_error_failure: i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.failure',
+    { defaultMessage: 'Unknown failure while executing command.' }
+  ),
+
+  // Dev:
+  // Max pending response zip uploads has been reached, limit 10. Endpoint can't use unlimited disk space.
+  'ra_execute_error_disk-quota': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.diskQuotaError',
+    {
+      defaultMessage: 'Too many pending command execution output zip files.',
+    }
+  ),
+
+  // Dev:
+  // The fleet upload API was unreachable (not just busy). This may mean policy misconfiguration, in which
+  // case health status in Kibana should indicate degraded, or maybe network configuration problems, or fleet
+  // server problems HTTP 500. This excludes offline status, where endpoint should just wait for network connection.
+  'ra_execute_error_upload-api-unreachable': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.uploadApiUnreachable',
+    {
+      defaultMessage:
+        'Failed to upload command execution output zip file. Unable to reach Fleet Server upload API.',
+    }
+  ),
+
+  // Dev:
+  // Perhaps internet connection was too slow or unstable to upload all chunks before unique
+  // upload-id expired. Endpoint will re-try a bit, max 3 times.
+  'ra_execute_error_upload-timeout': i18n.translate(
+    'xpack.securitySolution.endpointActionResponseCodes.execute.outputUploadTimeout',
+    {
+      defaultMessage: 'Failed to upload command execution output zip file. Upload timed out',
+    }
+  ),
+
+  // DEV:
+  // Upload API could be busy, endpoint should periodically re-try (2 days = 192 x 15min, assuming
```

Comment:
I'm curious what the 192 * 15min mean here for 2 days

Reply:
I think it's the logic that Endpoint uses. Maybe post that question to the internal tracking issue and I'm sure Leszek will clarify.